style.css

```diff
@@ -0,0 +1,118 @@
+header {
+  position: absolute;
+  width: 423px;
```

Comment:
- [ Optional ] It is not good practice to use fixed-sized width for it makes your website not responsive.

Reply:
What do you suggest to me to make it better for a responsive website? :))

Reply:
Hi @NomanRamzan 
thanks for asking. You can make it `width: 100%` for example. Then you can use padding on the container element if you need some space around.